src/joint-torque-controller.cpp

```diff
@@ -151,16 +165,50 @@ namespace dynamicgraph
       /* --- SIGNALS ---------------------------------------------------------- */
       /* --- SIGNALS ---------------------------------------------------------- */
 
+      DEFINE_SIGNAL_INNER_FUNCTION(dq_motor, dynamicgraph::Vector)
+      {
+        const Eigen::VectorXd& alpha              = m_velLeakingRateSIN(iter);
+        const Eigen::VectorXd& ddq_des_current    = m_jointsAccelerationsDesiredSIN(iter);
+        const Eigen::VectorXd& dq_current         = m_jointsVelocitiesSIN(iter);
+        const Eigen::VectorXd& ddq_des_prev       = buffer_ddq_des;
+        const Eigen::VectorXd& dq_prev            = buffer_dq;
+        Eigen::VectorXd& dq_motor                 = buffer_dq_motor;
+       
+        //Runge Kutta integration y:=dq_motor; dy=f(t,y); y(n) = y_n
```

Comment:
Why did you feel it was necessary to use a RK4 integration scheme? I think a straightforward Euler integration would do the job just as well, while making the code easier to read.